chutney/server/src/main/java/com/chutneytesting/security/infra/sso/SsoOpenIdConnectConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.security.infra.sso;
+
+public class SsoOpenIdConnectConfig {
+
+    public final String issuer;
+    public final String clientId;
+    public final String clientSecret;
+    public final String responseType;
+    public final String scope;
+    public final String redirectBaseUrl;
+    public final String ssoProviderName;
+    public final Boolean oidc;
+
+    public SsoOpenIdConnectConfig() {
```

Comment:
why this constructor ?

Reply:
This is for the default bean of SsoOpenIdConnectConfig, if sso in not activated

@Bean
    @ConditionalOnMissingBean
    public SsoOpenIdConnectConfig emptySsoOpenIdConnectConfig() {
        return new SsoOpenIdConnectConfig();
    }